media-gfx/freecad/freecad-0.16.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils fortran-2 python-single-r1 fdo-mime
+
+DESCRIPTION="QT based Computer Aided Design application"
+HOMEPAGE="http://www.freecadweb.org/"
+SRC_URI="https://github.com/FreeCAD/FreeCAD/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+#sci-libs/orocos_kdl waiting for Bug 604130 (keyword ~x86)
+#dev-qt/qtgui:4[-egl] and dev-qt/qtopengl:4[-egl] : Bug 564978 
+COMMON_DEPEND="dev-cpp/eigen:3
+	dev-libs/boost[python]
```

Comment:
make this `dev-libs/boost:=[python,${PYTHON_USEDEP}]` in order to enforce matching python impls